### ISSUE TITLE
We own @geralt-encore a beer!

### ISSUE THREAD

**Issue by artem-zinnatullin** [pushtorefresh/storio#732]
@geralt-encore dude, @nikitin-da and I own you a beer 🍻 for [all you've done for StorIO](https://github.com/pushtorefresh/storio/search?p=2&q=geralt-encore&type=Issues&utf8=✓)!

Ping us when you'll be in Saint-Petersburg (in 99% of cases I might be somewhere else, but I'll try my best to be there at that time) and we'll go to some bar and discuss many things 😸 

Maybe @nikitin-da will also invite some people from Yandex.Mail team :)

@nikitin-da thoughts?

**Comment by nikitin-da** [comment]
I completely agree! And I am certain this is bug, that we haven't done this yet! 🍺🍺🍺 
Sure I will invite guys!

**Comment by artem-zinnatullin** [comment]
Haha, true true, critical :D

**Comment by geralt-encore** [comment]
Thank's guys! It's a pleasure for me to work with you on StorIO 😊
Since now I am living in Helsinki so it's quite possible that eventually, I'll be in Saint-Petersburg and will be happy to have a beer with you 🍻
